dev/docker/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir -p /root/.ssh
 # Add deploy keys and SSH keys for Github with this container
 ADD dev/keys/deploy-wbia /root/.ssh/deploy-wbia
 
-ADD dev/keys/deploy-ibeis_cnn /root/.ssh/deploy-ibeis_cnn
+ADD dev/keys/deploy-wbia_cnn /root/.ssh/deploy-wbia_cnn
```

Comment:
We can remove all of the keys, we aren't using them anymore since we are using open source versions of these repos.

Reply:
Should I just remove the `dev/docker` stuff or may be leave those files unchanged?  The ones I worked on are in `devops/` and I'm not sure what to do with the docker files in `dev/docker`.

Reply:
I understand, I forgot we had an old Dockerfile in dev.  Yes, just delete the entire `dev/docker` folder.

Reply:
Ok deleted :+1: